Screenbox/Pages/HomePage.xaml

```diff
@@ -292,6 +293,16 @@
                     </VisualState.Setters>
                 </VisualState>
             </VisualStateGroup>
+
+            <VisualStateGroup x:Name="FlowDirectionStates">
+                <VisualState x:Name="LeftToRight" />
+
+                <VisualState x:Name="RightToLeft">
+                    <VisualState.Setters>
```

Comment:
Just nitpicking but I realized you can also define state trigger from XAML as well
```suggestion
                    <VisualState.StateTriggers>
                        <StateTrigger IsActive="{x:Bind class:App.IsRightToLeftLanguage}" />
                    </VisualState.StateTriggers>
```